data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/handlers/AbstractHandler.java

```diff
@@ -28,6 +29,15 @@ protected AbstractHandler(final JsonProvider jsonProvider) {
     this.jsonProvider = jsonProvider;
   }
 
+  public <T> T parseRequestBody(String json, Class<T> clazz) {
```

Comment:
very small nit: protected?
```suggestion
  protected <T> T parseRequestBody(String json, Class<T> clazz) {
```